integration-tests/src/test/java/org/apache/fineract/integrationtests/DelinquencyActionIntegrationTests.java

```diff
@@ -47,9 +47,9 @@
 import org.apache.fineract.client.models.PostLoansDelinquencyActionResponse;
 import org.apache.fineract.client.util.CallFailedRuntimeException;
 import org.apache.fineract.integrationtests.common.ClientHelper;
-import org.apache.fineract.integrationtests.common.loans.LoanTestLifecycleExtension;
 import org.apache.fineract.integrationtests.common.products.DelinquencyBucketsHelper;
 import org.apache.fineract.integrationtests.inlinecob.InlineLoanCOBHelper;
+import org.apache.fineract.integrationtests.sumas.loans.LoanTestLifecycleExtension;
```

Comment:
The package name change is bound to raise many conflicts if there is any upgrade from community, is this absolutely necessary?

Reply:
I needed to have a package to keep only only the tests specific the sumas
Allow me to review a better approach which would favor future upgrade. Perhaps I can create a separate

Reply:
A separate module might suit this better i think.

Reply:
Great, let me fix that 
thanks